### PR TITLE
Add gambling icons

### PR DESCRIPTION
**Pull request by dvdmrtnz**
* Add `amenity=gambling`, `leisure=adult_gaming_centre`, `leisure=amusement_arcade`, `shop=bookmaker` and `shop=lottery` icons. Closes https://github.com/organicmaps/organicmaps/issues/210

| Day | Night |
|-|-|
|  |  |
|  |  | 
|  |  |

### CODE REVIEW

**Review by vng — APPROVED** [review]

File: data/categories.txt [hunk 1]
```diff
@@ -4538,6 +4538,123 @@ sk:Kasíno
 sw:Kasino|burudani
 fa:قمارخانه|کازینو
 
+amenity-gambling
+en:Gambling
```
Comment (vng) [inline data/categories.txt:4542]: Probably we should add separate category ```@gambling``` and assign it on casino, bookmaker, gambling, adult_gaming

Comment (vng) [reply data/categories.txt:4542]: Want to see all the "trash" with one query :)

Comment (biodranik) [reply data/categories.txt:4542]: Not in the app's UI, please. Let it remain in search-queries only.

Comment (vng) [reply data/categories.txt:4542]: Of course

Comment (dvdmrtnz) [reply data/categories.txt:4547]: Added category to search

File: data/categories.txt [hunk 2]
```diff
@@ -4504,7 +4504,85 @@ sk:Nevestinec|verejný dom
 sw:Danguro
 fa:فاحشه خانه|جنده خانه
 
-amenity-casino|@entertainment|@nightlife
+@gambling
+en:4Gambling
```
Comment (vng) [inline data/categories.txt:4508]: Remove 4, do not suggest it, search only for full name.

And I'd also remove all digits in casino, bookmaker, lottery, etc, if any.

Comment (dvdmrtnz) [reply data/categories.txt]: Done

**Pull request merged**